packages/sync/src/class-actions.php

```diff
@@ -63,8 +70,11 @@ class Actions {
 	 *
 	 * @access public
 	 * @static
+	 * @param \Jetpack $jetpack the main Jetpack class object.
 	 */
-	public static function init() {
+	public static function init( \Jetpack $jetpack ) {
```

Comment:
Yet again, I'd love it if we don't introduce such references to Jetpack in packages, because this makes the package dependent on Jetpack itself, which we always wanted to avoid.

Reply:
Thank you for your input, I have replied earlier.